insights/parsers/krb5.py

```diff
@@ -47,62 +50,104 @@
 
 from insights.core import Parser
 from insights.core.plugins import parser
+from .. import get_active_lines
 
 
-@parser('krb5')
-@parser('krb5.conf.d')
-class Krb5Configuration(Parser):
-    """Class for krb5.conf and krb5.conf.d configuration."""
-
+class Krb5ConfigParser(Parser):
     def parse_content(self, content):
         dict_all = {}
         is_squ = False
         section_name = ""
         squ_value = {}
         squ_section_name = ""
         section_value = {}
-        for line in content:
-            """Get active line """
-            if "includedir" in line or line.startswith("#") or not line:
+        for line in get_active_lines(content):
+            if "includedir" in line:
```

Comment:
Better add one line:
~~~
line = line.strip()
~~~

then use it in below code.